### PR TITLE
[infomanager] Move Sysinfo out of infomanager

### PR DESCRIPTION
**Pull request by Paxxi**
This is a small start of breaking up infomanager and getting rid of functions it shouldn't be responsible for.
Moved version information into our sysinfo class
moved info labels to their own header to be able to include those without all of infomanager. This is useful for providers, as they need to know about the labels but have no need for infomanager and it avoids some cyclic dependencies.
cleaned up some includes because of this.

### CODE REVIEW

File: xbmc/addons/Visualisation.cpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 #include "Visualisation.h"
 #include "utils/fft.h"
 #include "GUIInfoManager.h"
```
Comment (da-anda) [inline xbmc/addons/Visualisation.cpp]: wouldn't you have to drop this line just like in GUIDialogProgress below?

Comment (Paxxi) [inline xbmc/addons/Visualisation.cpp]: I do the same 😊

File: xbmc/guilib/GUIControlGroupList.cpp [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 #include "GUIControlGroupList.h"
 #include "input/Key.h"
-#include "GUIInfoManager.h"
+#include "guiinfo/GUIInfoLabels.h"
```
Comment (xhaggi) [inline xbmc/guilib/GUIControlGroupList.cpp]: As we the use CGUIInfoLabels heavily in our guilib why not move it there instead of the new `guiinfo` folder.

Comment (da-anda) [inline xbmc/guilib/GUIControlGroupList.cpp]: I think the plan is to move more and more stuff out from GUIInfoManager into dedicated InfoProvider classes. So a info provider class for player related stuff, add-on related, pvr related, ...

Comment (Paxxi) [inline xbmc/guilib/GUIControlGroupList.cpp]: What @da-anda said and it's not really part of guilib even if it's heavily used there

**Comment by Paxxi** [comment]
jenkins build this please

**Comment by Paxxi** [comment]
@mkortstiege or @Memphiz can you fix the xcode project?

**Comment by mkortstiege** [comment]
@Paxxi, please pick https://github.com/mkortstiege/xbmc/commit/fee26614c0e73746783d79e3901d6cfa98d477be

File: xbmc/epg/GUIEPGGridContainer.cpp [hunk 3]
```diff
@@ -1295,7 +1299,7 @@ void CGUIEPGGridContainer::SetSelectedChannel(int channelIndex)
 {
   if (channelIndex < 0)
     return;
-
+
```
Comment (MartijnKaijser) [inline xbmc/epg/GUIEPGGridContainer.cpp]: Some white spaces were added in this file

**Pull request merged**